AmplifyPlugins/Core/AWSPluginsCoreTests/Model/Decorator/AuthRuleDecorator/ModelWithOwnerAuthAndGroupWithGroupClaim.swift

```diff
@@ -13,63 +13,63 @@ import XCTest
 
 /*
  type OIDCGroupPost
-   @model
-   @auth(
-     rules: [
-       { allow: owner, provider: oidc, identityClaim: "sub"},
-       { allow: groups, provider: oidc, groups: ["Admins"],
-                                    groupClaim: "https://myapp.com/claims/groups"}
-     ]
-   ) {
-   id: ID!
-   title: String!
-   owner: String
+ @model
+ @auth(
+ rules: [
+ { allow: owner, provider: oidc, identityClaim: "sub"},
+ { allow: groups, provider: oidc, groups: ["Admins"],
+ groupClaim: "https://myapp.com/claims/groups"}
+ ]
+ ) {
+ id: ID!
+ title: String!
+ owner: String
```

Comment:
nit-picking: leave it formatted (I usually use Prettier playground for GraphQL formatting: https://prettier.io/playground/).